src/platforms/python/guides/rq/index.mdx

```diff
@@ -31,6 +31,24 @@ The integration will automatically report errors from all RQ jobs.
 
 Generally, make sure that the **call to `init` is loaded on worker startup**, and not only in the module where your jobs are defined. Otherwise, the initialization happens too late and events might end up not being reported.
 
+In addition, make sure that **`init` is called once** in your app and imported from there in the different places it is needed. In a Flask app, for example, Sentry and `RqIntegration` can be initalised as 
+
+```python {filename:app.py}
+import sentry_sdk
+from sentry_sdk.integrations.flask import FlaskIntegration
+from sentry_sdk.integrations.rq import RqIntegration
+
+sentry = sentry_sdk.init(
+    dsn=PUBLIC_DSN,
```

Comment:
```suggestion
    dsn=___PUBLIC_DSN___,
```

Need the `___` to get it magically replaced with a valid DSN when reading docs.sentry.io.